docs/ru/extended_roadmap.md

```diff
@@ -429,13 +435,13 @@ Upd. В рамках данной задачи добавляем подстве
 
 Как в Аркадии, но только в качестве fallback.
 
-### 7.17. Доделать tgz пакеты.
+### 7.17. + Доделать tgz пакеты.
 
 Уже давно собираются универсальные tgz пакеты, но по нелепой случайности из них исчез install скрипт.
 [Александр Сапин](https://github.com/alesapin). Может делегировать эту задачу кому угодно.
 Upd. Сделано всё кроме инструкции на сайте. Для этого требуется создать директории testing/stable/prestable на repo.yandex.ru. Внезапно оказалось, что человек, отвечающий за это, в отпуске, и он не отвечает на вопрос, кто его заместитель. Q1.
 
-### 7.18.1. Доделать бинарники под Mac.
+### 7.18.1. + Доделать бинарники под Mac.
```

Comment:
Not ready.
There is no instruction on the main https://clickhouse.tech/ page neither in documentation.